src/components/container/Container.js

```diff
@@ -34,13 +39,18 @@ export class ContainerComponent extends FormioComponents {
     if (!value || !_.isObject(value)) {
       return;
     }
+    this.data[this.component.key] = value;
```

Comment:
This will probably break the "changed" trigger from occurring.

Reply:
This one is necessary since Components.updateValue never sets the original data object for the component. Sub fields will still trigger changed.